odoo/models.py

```diff
@@ -170,14 +170,18 @@ def fix_import_export_id_paths(fieldname):
 def to_company_ids(companies):
     if isinstance(companies, BaseModel):
         return companies.ids
-    elif isinstance(companies, (list, tuple)):
+    elif isinstance(companies, (list, tuple, str)):
```

Comment:
Shouldn't we return `[companies]` if `companies=='company_id'`?
This method is supposed to return a list of companies

For instance, when we do this below, we shouldn't end up with `['c', 'o', 'm', 'p', 'a', 'n', 'y', '_', 'i', 'd']`
```python
companies = [id for id in to_company_ids(companies) if id]
```